scripts/generate_can_enums.py

```diff
@@ -55,7 +55,10 @@ def canid_filters_formatted(canid_filters):
 
 
 def roboteq_enums_formatted(roboteq_enums):
-    return ''.join(['\t{} = {},\n'.format(roboteqid_name, hex(roboteqid_value).upper().replace('X', 'x')) for roboteqid_value, roboteqid_name in sorted(roboteq_enums.items())])
+    if roboteq_enums:
+        return ''.join(['\t{} = {},\n'.format(roboteqid_name, hex(roboteqid_value).upper().replace('X', 'x')) for roboteqid_value, roboteqid_name in sorted(roboteq_enums.items())])
```

Comment:
prefer using python format strings instead of old style `.format`

Reply:
Updated but what's wrong with `.format`?

Reply:
`.format` is a lot less readable. f-strings let u read the values inline (assuming ur variables are named well). 

more here: https://realpython.com/python-f-strings/